sdk/jobs/automl-standalone-jobs/automl-image-classification-multiclass-batch-scoring/image-classification-multiclass-batch-scoring-non-mlflow-model.ipynb

```diff
@@ -0,0 +1,842 @@
+    "    \n",
+    "- **The environment** in which the model runs. It can be a Docker image with Conda dependencies, or an environment already registered in your workspace referenced using `azureml:<environment-name>:<environment-version>`.  In this notebook we will create a environment object using the environment definition downloaded from the best_child_run artifacts.\n",
+    "\n",
+    "- **The pre-created compute** referenced using `azureml:<compute-name>`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 5.1 Regsiter The Model in the workspace\n",
+    "For deploying the model, we first need to register it with the existing workspace so that we can discover it during runtime."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml.entities import Model\n",
+    "\n",
+    "model_name = \"fridge-items-model\"\n",
+    "model = Model(\n",
+    "    path=f\"azureml://jobs/{best_run.info.run_id}/outputs/artifacts/outputs/model.pt\",\n",
+    "    name=model_name,\n",
+    "    description=\"my sample object detection model\",\n",
+    ")\n",
+    "\n",
+    "# for downloaded file\n",
+    "# model = Model(path=\"artifact_downloads/outputs/model.pt\", name=model_name)\n",
+    "\n",
+    "registered_model = ml_client.models.create_or_update(model)\n",
+    "registered_model.id"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 5.2 Configure environment\n",
+    "It is recommended to use the same envionment for model deployment as the model training, therefore we are creating the Environment object using the conda environment file downloaded as artifacts. We also need to specify the base image, which in case of vision tasks, is `mcr.microsoft.com/azureml/openmpi4.1.0-cuda11.1-cudnn8-ubuntu18.04`\n",
+    "\n",
+    "To read more about environments, please follow this [notebook](./../../../assets/environment/environment.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml.entities import Environment\n",
+    "\n",
+    "env = Environment(\n",
+    "    name=\"automl-images-env\",\n",
+    "    description=\"environment for automl images inference\",\n",
+    "    image=\"mcr.microsoft.com/azureml/openmpi4.1.0-cuda11.1-cudnn8-ubuntu18.04\",\n",
+    "    conda_file=\"artifact_downloads/outputs/conda_env_v_1_0_0.yml\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 5.3 Get A Scoring Script\n",
+    "\n",
+    "To do the scoring, you need tp create a batch scoring script batch_scoring.py, and write it to the scripts folder in current directory. The script takes a minibatch of input images, applies the classification model, and outputs the predictions to a results file.\n",
```

Comment:
Sorry Phani, I forgot to address this. The model_name is not required for this deployment. It was required in the v1 notebook since we were using the pipelines and when we register the model in the pipeline, it changes the filename from 'model.pt' to 'best_model_data'(basically without .pt extension). So to handle that case, we pass the model_name as the parameter to the pipeline and find the model by `Model.get_model_path(args.model_name), "model.pt")`. 

In the pipelines notebook using v2, we might require the same unless the azureml-pipelines has changed the way they pass the model between steps or we deploy directly the mlflow model.